unic/normal/src/decompose.rs

```diff
@@ -13,17 +13,18 @@
 use std::fmt::{self, Write};
 
 use unic_ucd_normal::{CanonicalCombiningClass, decompose_canonical, decompose_compatible};
+use unic_ucd_normal::canonical_combining_class::values as ccc;
 
 
 // Helper functions used for Unicode normalization
-fn canonical_sort(comb: &mut [(char, u8)]) {
+fn canonical_sort(comb: &mut [(char, CanonicalCombiningClass)]) {
     let len = comb.len();
     for i in 0..len {
         let mut swapped = false;
         for j in 1..len - i {
             let class_a = comb[j - 1].1;
             let class_b = comb[j].1;
-            if class_a != 0 && class_b != 0 && class_a > class_b {
+            if class_a != ccc::NotReordered && class_b != ccc::NotReordered && class_a > class_b {
```

Comment:
I say 👍 to that.

As a side note, should `is(_not)_reordered` be `#[inline]`? It's [somewhat controversial](https://stackoverflow.com/q/37639276/3019990) but this seems like the small-enough-that-we-want-it-to-be-inline-across-crates kind of thing.